tools/batchTestGenerator/internal/mock_test_data/eksoperatorarm64test.json

```diff
@@ -7,11 +7,7 @@
                 {
 		    "case_name": "prometheus_static_adot_operator",
 		    "platforms": ["EKS_ADOT_OPERATOR", "EKS_ADOT_OPERATOR_ARM64"]
-		},
-                {
-                    "case_name": "prometheus_sd_adot_operator_awsprw",
-                    "platforms": ["EKS_ADOT_OPERATOR", "EKS_ADOT_OPERATOR_ARM64"]
-                },
+        		},
```

Comment:
Can you also fix the spacing for L8-10 here, it seems to be tabs instead of spaces